### DISCUSSION TITLE
Is there any example on how to "Sequential Recommendations"?

### DISCUSSION THREAD

**Discussion by callmeBalloch** [NVIDIA-Merlin/Transformers4Rec#384] [Q&A]
Hi guys!
Really pogs for the repo, I think is awesome.
Anyway I noted that all the documentation and examples are focused on the topic of "Season-based" recommendation, with a lack of simple and useful examples about "Sequential Recommendations".
Is there anything that could help with a bare minimum setup?

**Comment by Folahan07** [comment]
I wonder why
